### PR TITLE
Evidence label Normalization

### PR DESCRIPTION
**Pull request by mayfielg**
Previously, 'evidence label' amalgamated two different types of information, depending on the source (civic, jax, etc.). It had either response type information (sensitive, resistant) or clinical status stage information (Phase 1, preclinical, FDA-approved) or both. 

A new data label has been created in 'association' to parse apart these two types of data. 'response_type' now refers to the resistance or sensitivity a g2p association, if known, and 'evidence_label' now refers to the clinical status of the association, if known. 

Based on the current dashboard, this puts clinical data on the table in kibana. If we want response_type to be present as well, we'll need to alter the dashboard. 

A few of the data sources do not provide any sort of obvious observation of one or either of these data types; if this is the case, minor stripping of information from the given description is applied.

### CODE REVIEW

File: harvester/jax.py [hunk 1]
```diff
@@ -105,7 +109,18 @@ def convert(jax_evidence):
             }
         }]
         # add summary fields for Display
-        association['evidence_label'] = evidence['response_type']
+        for item in match.ev_lab:
```
Comment (bwalsh) [inline harvester/jax.py]: @mayfielg Thanks for the PR.  Looks pretty great.
Two requests:
* can you open it against v0.3?
* would it be a good idea to collapse the repeated block of code here into a method in match?  The result would be a single line calling the new method in each of the harvesters.

Comment (mayfielg) [inline harvester/jax.py]: Addressed these concerns. Thoughts?

Comment (bwalsh) [inline harvester/jax.py]: @mayfielg 
Can you submit the PR against v0.3?

**Comment by bwalsh** [comment]
Thanks.  Running this now.

**Comment by bwalsh** [comment]
@mayfielg made a few tweaks, please review

**Comment by grmayfie** [comment]
Looked through, I also had that error and resolved it the same way. Deleting this branch as it is fully merged and unnecessary now.
